README.md

```diff
@@ -1,2 +1,29 @@
-# labels
-Copies labels into a new repo
+# 🏷 Labels
+Copies labels from one repo to another
+
+
+## Why?
+Having a standard list of labels across all of your projects means that you can
+move between repositories quickly and effectively. *However*, manually adding
+labels to a new GitHub repository can become old, ***fast***. This module aims
+to **save you time** by automating the addition of labels to a new project by
+copying them from a selected repo and then transferring them to a target repo.
+
+Our main criteria is that it ***MUST*** be quicker than manually adding labels.
+
+## What?
+A quick and easy way to add an existing list of standardised labels to a GitHub
+repo.
+
+## Who?
+#### Who should use this?
+Any developer who has created multiple repositories and values their time ⏰
+
+## How?
+A simple and intuitive UI authenticated with GitHub. It will include the
+following fields:
+
+* repo that you want to copy labels from (any public repo)
+* list of target repos that you want to add labels to (one that you have access to)
+* GitHub username
+* GitHub password
```

Comment:
under _no_ circumstances can we ask people for their Password... 😕 
(_or are you referring to the GitHub OAuth Page...?_)